frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -0,0 +1,146 @@
+      <h2 className={classes.subHeader}>Beskrivelse</h2>
+      <p className={classes.text}>
+        Her må du beskrive tjenesten. Teksten kan bli synlig på flere områder på tvers av offentlige
+        nettløsninger.
+      </p>
+      <p className={classes.subTitle}>{'Bokmål (standard)'}</p>
+      <div className={classes.inputWrapper}>
+        <TextArea
+          value={description}
+          resize='vertical'
+          placeholder='Tekst'
+          onChange={(e) => setDescription(e.currentTarget.value)}
+          rows={5}
+          aria-label='Beskrivelse'
+        />
+      </div>
+      <h2 className={classes.subHeader}>Hjemmeside</h2>
+      <p className={classes.text}>Link til nettsiden der tjenesten kan startes av brukeren.</p>
+      <div className={classes.inputWrapper}>
+        <TextField
+          value={homepageURL}
+          onChange={(e) => setHomepageURL(e.target.value)}
+          aria-label='Hjemmeside'
+        />
+      </div>
+
+      <h2 className={classes.subHeader}>Nøkkelord</h2>
+      <p className={classes.text}>
+        {'Skriv nøkkelord for ressursen, separer hvert ord med et komma ","'}
+      </p>
+      <div className={classes.inputWrapper}>
+        <TextField
+          value={keywords}
+          onChange={(e) => setKeywords(e.target.value)}
+          aria-label='Skriv nøkkelord for ressursen, separer hvert ord med et komma ","'
+        />
+      </div>
+      <h2 className={classes.subHeader}>Velg sektor for ressursen</h2>
+      {/* TODO - inform user that this is optional */}
+      <div className={classes.inputWrapper}>
+        <Select
+          multiple
+          options={resourceSectorsMockOptions}
+          onChange={(e) => setSectors(e)}
+          value={sectors}
+          label='Velg sektor for ressursen'
+          hideLabel
+        />
+      </div>
+      <h2 className={classes.subHeader}>Velg tematisk område</h2>
+      <div className={classes.inputWrapper}>
+        <Select
+          multiple
+          options={resourceThematicAreaMockOptions}
+          onChange={(e) => setThematicArea(e)}
+          value={thematicArea}
+          label='Velg tematisk område'
+          hideLabel
+        />
+      </div>
+      <h2 className={classes.subHeader}>Delegasjonstekst</h2>
+      <div className={classes.inputWrapper}>
+        <TextField
+          value={delegationText}
+          onChange={(e) => setDelegationText(e.target.value)}
+          aria-label='Delegasjonstekst'
+        />
+      </div>
+      <h2 className={classes.subHeader}>Vis i offentlige kataloger</h2>
+      <p className={classes.text}>
+        Etter publisering blir ressursen tilgjengelig i kataloger, blant annet i altinn, på norge.no
+        og data.norge.no.
+      </p>
+      <div className={classes.inputWrapper}>
+        <Switch isChecked={showInPublic} onToggle={(b: boolean) => setShowInPublic(b)} />
+        <p
+          className={showInPublic ? classes.toggleTextActive : classes.toggleTextInActive}
```

Comment:
```suggestion
          className={showInPublic ? classes.toggleTextActive : classes.toggleTextInactive}
```
Keep inactive as one word for both grammar purposes and easier readability